### ISSUE TITLE
RGB Color for Sleep Mode

### ISSUE THREAD

**Issue by fabianschaefer** [claytonjn/hass-circadian_lighting#83]
I really like your component, great work, thanks a lot. Would it be possible to add a feature to set an RGB value instead of sleep_colortemp. I prefer blue light in sleep mode because it wakes me up less. 
By the way, have you thought about making this an official HASS Integration or integrate it into the existing Flux component that does the same witch much less features.

**Comment by Agratam** [comment]
I agree on the blue light.
It's actually the reason why I currently disable the component at night and I define a "manual" scene to be used.
Matteo

**Comment by gitrichhub** [comment]
I agree on the suggestion.  I use this awesome function for an aquarium light and it would be nice to have it transition to a blue color after sunset.  Especially if the transition could be gradual.

**Comment by maichai** [comment]
+1 for sleep_RGB, not just for blue, but any RGB value. Actually red is the easiest to they eyes to go back to sleep.
